packages/glimmer-runtime/lib/dom/change-lists.ts

```diff
@@ -57,38 +57,54 @@ export function defaultAttributeChangeLists(tagName: string, attr: string) {
 }
 
 export function readDOMAttr(element: Element, attr: string) {
-   let isSVG = element.namespaceURI === SVG_NAMESPACE;
-   let { type, normalized } = normalizeProperty(element, attr);
-
-   if (isSVG) {
-     return element.getAttribute(normalized);
-   }
-
-   if (type === 'attr') {
-     return element.getAttribute(normalized);
-   } {
-     return element[normalized];
-   }
+  let isSVG = element.namespaceURI === SVG_NAMESPACE;
+  let { type, normalized } = normalizeProperty(element, attr);
+
+  if (isSVG) {
+    return element.getAttribute(normalized);
+  }
+
+  if (type === 'attr') {
+    return element.getAttribute(normalized);
+  } {
+    return element[normalized];
+  }
 };
 
 export const PropertyChangeList: IChangeList = {
   setAttribute(env: Environment, element: Simple.Element, attr: string, value: Opaque, namespace?: DOMNamespace) {
     if (value !== null && value !== undefined) {
       let normalized = attr.toLowerCase();
-      element[normalized] = normalizePropertyValue(value); // TODO: This doesn't work
+      element[normalized] = value;
```

Comment:
There are some assertions in Ember around `<input disabled>` that we need to add to the tests below.  

If a template contains any of the following:

* `<input disabled=''>`
* `<input disabled>`
* `<input disabled="true">`
* `<input disabled="false">`

The `disabled` property needs to result in `true`, but given the logic here ATM I believe it would not be (happy to be proven wrong by way of added tests for this below).

![screenshot of DOM stuffs](https://monosnap.com/file/nYkSB9VMmpu4U10U1YQEcMvApJPpxa.png)

However, if the template contains any of these we should end up with `false` (I believe the `null` and `undefined` cases are handled currently):

* `<input disabled={{somethingThatEqualsFalse}}>`
* `<input disabled={{somethingUndefined}}>`
* `<input disabled={{somethingNull}}>`